core/lib/presentation/views/html_viewer/html_content_viewer_widget.dart

```diff
@@ -183,6 +184,18 @@ class _HtmlContentViewState extends State<HtmlContentViewer> {
       if (scrollWidth != null && offsetWidth != null && mounted) {
         final isScrollActivated = scrollWidth.round() == offsetWidth.round();
         log('_HtmlContentViewState::_setActualWidthView():isScrollActivated: $isScrollActivated');
+        if (isScrollActivated) {
```

Comment:
i think `horizontalGestureActivated = !isScrollActivated;` is more readable.

Reply:
I also perform another action in this condition. You can write as you please, but we'll need to write more conditions to handle the next action.